src/Jwt/Udb3Token.php

```diff
@@ -64,4 +64,16 @@ public function audienceContains(string $audience): bool
 
         return in_array($audience, $aud, true);
     }
+
+    public function canUseEntryAPI(): bool
+    {
+        $apis = $this->token->getClaim('https://publiq.be/publiq-apis', '');
```

Comment:
Is the difference between the claim name and the metadata name intentional?
https://github.com/cultuurnet/udb3-search-service/blob/main/src/Http/Authentication/AuthenticateRequest.php#L151

Reply:
Custom claims in Auth0 always have to be namespaced as an URL. We could also namespace the metadata of course, but there it's not required and maybe overkill? I have no strong opinion on it but if you think we should change it we'd have to change the metadata check in SAPI3 of course.

Reply:
See also https://auth0.com/docs/tokens/create-namespaced-custom-claims

Reply:
No it's perfect to keep it as is. Just wanted to make sure there is no correlation between those "key" values.